crates/evm/networks/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+//! # foundry-evm-precompiles
+//!
+//! Foundry EVM network custom precompiles.
+
+use crate::celo::transfer::{CELO_TRANSFER_ADDRESS, PRECOMPILE_ID_CELO_TRANSFER};
+use alloy_evm::precompiles::PrecompilesMap;
+use alloy_primitives::Address;
+use clap::Parser;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+
+pub mod celo;
+
+#[derive(Clone, Debug, Default, Parser, Copy, Serialize, Deserialize)]
+pub struct NetworkConfigs {
+    /// Enable Optimism network features.
+    #[arg(help_heading = "Networks", long, visible_alias = "optimism")]
+    #[serde(skip)]
```

Comment:
why is this serde skipped?

Reply:
this ends up in forge config and we don't have (yet) anything to add to forge if optimism set to true, hence decided not to include